org.adempiere.base/src/org/compiere/print/DataEngine.java

```diff
@@ -1185,14 +1196,25 @@ else if (pdc.getDisplayType() == DisplayType.DateTime)
 									valueString = DisplayType.getDateFormat(pdc.getDisplayType(), m_language).format(value);
 								if (format.getTableFormat().isPrintFunctionSymbols())		//	Translate Sum, etc.
 									valueString	+= PrintDataFunction.getFunctionSymbol(functions[f]);
+								if (m_group.isFunctionColumn(pdc.getAD_PrintFormatItem_ID(), functions[f])) {
+									Object groupValue = m_group.getValue(pdc.getAD_PrintFormatItem_ID(), 
+											pdc.getAD_PrintFormatItem_ID(), functions[f]);
+									if(groupValue instanceof Timestamp)
+										groupValue = formatTimestamp((Timestamp)groupValue);
```

Comment:
I think should format with pdc.getDisplayType here (i.e it could be just date or date+time). same for other place that call formatTimestamp.